index.js

```diff
@@ -1,13 +1,39 @@
 class SortedList {
-  constructor() {}
+  
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+     this.length++;
+   return this.items.sort(function(a, b){
+   return a - b;
+  })
+   return this.items;
+}
+
+  get(pos) {
+    if (pos.this.length && !this.length){
```

Comment:
pos has no attribute `this`. You should check if `pos` (a number) is lesser than the items array's length